apps/condo/pages/_app.tsx

```diff
@@ -62,59 +63,79 @@ const ANT_LOCALES = {
     en: enUS,
 }
 
-interface IMenuItemData {
-    path: string,
-    icon: React.FC,
-    label: string,
-}
-
-const ANT_DEFAULT_LOCALE = enUS
-
-const MenuItems: React.FC = () => {
-    const { link } = useOrganization()
-    const { isExpired } = useServiceSubscriptionContext()
-    const hasSubscriptionFeature = hasFeature('subscription')
-    const disabled = !link || (hasSubscriptionFeature && isExpired)
-    const { isCollapsed } = useLayoutContext()
-    const { wrapElementIntoNoOrganizationToolTip } = useNoOrganizationToolTip()
-
-    const menuItemsData: IMenuItemData[] = [{
+const itemsConfigs = [
+    {
         path: 'reports',
         icon: BarChartIconNew,
         label: 'menu.Analytics',
+        access: ['canGetExternalReportIframeUrl'],
     }, {
         path: 'ticket',
         icon: BarTicketIcon,
         label: 'menu.ControlRoom',
+        access: ['canReadTickets'],
     }, {
         path: 'property',
         icon: BarPropertyIcon,
         label: 'menu.Property',
+        access: ['canReadProperties'],
     }, {
         path: 'contact',
         icon: BarUserIcon,
         label: 'menu.Contacts',
+        access: ['canReadContacts'],
     }, {
         path: 'employee',
         icon: BarEmployeeIcon,
         label: 'menu.Employees',
+        access: ['canReadOrganizationEmployees'],
     }, {
         path: 'billing',
         icon: BarBillingIcon,
         label: 'menu.Billing',
+        access: ['canReadBillingReceipts'],
     }, {
         path: 'payments',
         icon: BarPaymentsIcon,
         label: 'menu.Payments',
+        access: ['canReadPayments'],
     }, {
         path: 'meter',
         icon: BarMeterIcon,
         label: 'menu.Meters',
+        access: ['canReadMeters'],
```

Comment:
we don't have can Read access for models. It's not so easy. it's not a boolean

Reply:
In general it should be a function.

Reply:
It's not about models. It's about roles and which of them can see menu items.

Reply:
We don't have such access in roles. And it's not enough to check true/false